.eslintrc.js

```diff
@@ -44,7 +44,8 @@ module.exports = {
     "jsx-a11y/label-has-for": "warn", // see #173
     "no-console": "warn",
     "no-extra-semi": "off", // because it isn't that important
-    "react/prop-types": "warn" // see #175
+    "react/prop-types": "warn", // see #175
+    "jsx-a11y/no-onchange": "warn"
```

Comment:
did you look at this? https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md (I did a google search on "jsx-a11y/no-onchang"). Will adding or substituting onBlur be acceptable here, instead of adding a rule exception?

Reply:
👌 It's not throwing an error anymore, so I removed the rule.